scripts/tts_engines/eleven_labs.py

```diff
@@ -0,0 +1,27 @@
+import requests
+from config import Config
+cfg = Config()
+
+def narrate(input_script, output_dir):
+    output_path = output_dir + "/narration.mpeg"
+
+    tts_headers = {
+        "Content-Type": "application/json",
+        "xi-api-key": cfg.elevenlabs_api_key
+    }
+
+    tts_url = "https://api.elevenlabs.io/v1/text-to-speech/{voice_id}".format(
+        voice_id="ErXwobaYiN019PkySvjV")
```

Comment:
I think it makes sense to also extract out the settings for which voice on the engine you want to use if you are adding this option. We can have a default but it makes sense to be able to specify one

Reply:
I think that's a good feature but one for the TODO rather than this PR

Reply:
The video script isn't supposed to be a settings file for the app it's supposed to describe the video so it feels like it should go in this PR. I could go and change the voice settings in the app for the google engine. I think it's misleading unless those settings are added.

Reply:
To me it's further functionality, we don't support other voices yet - just using different engines.

There are use cases where a longer video might want to use the cheaper engine for most, and the more expensive engine for some other parts where quality matters more, such as a long educational video with summaries.

Other voices is a key feature, likely up next but not what I'm adding in these changes.

Reply:
The point of the upscale mode was so we could change the settings with CLI args. If you just want to change the engine then it should be using something like that.

Reply:
In a queue based system we can't use different args each time we receive a message, the yaml being a single source of truth to specify what we generate makes the most sense to me, with some flags we can use for debugging.

Perhaps let's think this over and discuss later? The runtime args vs input settings question is an interesting and important one.